docs/usage-notes.md

```diff
@@ -1,4 +1,6 @@
 - [Get Started](#get-started)
+- [Notes on Using the Desktop App](#notes-on-using-the-web-app)
```

Comment:
Looks like there are typos in the headings?

Reply:
Thanks for pointing it out, I must have missed it!